src/async_fuse/memfs/dist/etcd.rs

```diff
@@ -1,6 +1,7 @@
 use crate::async_fuse::memfs::INum;
 use crate::common::error::Context;
 use crate::common::etcd_delegate::EtcdDelegate;
+use datenlord::common::etcd_delegate::KVVersion;
```

Comment:
I prefer `KvVersion` instead of `KVVersion`.

Reply:
OK.